### DISCUSSION TITLE
Update to .NET6 and add native macOS ARM64 builds

### DISCUSSION THREAD

**Discussion by collinkreutzer** [ppy/osu#15765] [Ideas] [answered]
As .NET6 now supports macOS ARM64 (Apple Silicon), this shouldn't be too difficult, i guess? Sadly I do not know anything about .NET and couldn't get it to run by myself for now. After updating .NET and changing the runtime to osx.11.0-arm64, the game would throw the following exception before crashing: 
`System.DllNotFoundException: Unable to load shared library 'realm-wrappers' or one of its dependencies.`

**Comment by bdach** [comment, marked as answer]
please use the search functionality

https://github.com/ppy/osu/issues/12210
https://github.com/ppy/osu/discussions/15544

also you are most likely getting `DllNotFoundException` because realm is not shipping `osx-arm64` shared libs, so we're dependent on upstream for supporting arm64 natively in that respect
